frontends/web/src/components/TaskOwnerPageComponents/Settings.js

```diff
@@ -204,6 +215,70 @@ const Settings = (props) => {
                           />
                         </Col>
                       </Form.Group>
+                      <Form.Group
+                        as={Row}
+                        controlId="build_sqs_queue"
+                        className="py-3 my-0 border-bottom"
+                      >
+                        <Form.Label column>
+                          Build queues
+                          <Form.Text id="paramsHelpBlock" muted>
+                            build queue
+                          </Form.Text>
+                        </Form.Label>
+                        <Col sm="6">
+                          <Form.Control
+                            type="text"
+                            defaultValue={values.build_sqs_queue}
+                            onChange={handleChange}
+                          />
+                        </Col>
+                      </Form.Group>
+                      <Form.Group
+                        as={Row}
+                        controlId="eval_sqs_queue"
+                        className="py-3 my-0 border-bottom"
+                      >
+                        <Form.Label column>
+                          Eval queues
+                          <Form.Text id="paramsHelpBlock" muted>
+                            eval queue
+                          </Form.Text>
+                        </Form.Label>
+                        <Col sm="6">
+                          <Form.Control
+                            type="text"
+                            defaultValue={values.eval_sqs_queue}
+                            onChange={handleChange}
+                          />
+                        </Col>
+                      </Form.Group>
+                      <Form.Group
+                        as={Row}
+                        controlId="is_decen_task"
+                        className="py-3 my-0 border-bottom"
+                      >
+                        <Form.Label column>
+                          Is this a Decentralized Task?
```

Comment:
I did this b/c douwe suggested I add this as a field - I think it is because we wanted to not disrupt Dynabench  but still be able to test decentralized eaas in prod (so legitimate task owners outside still can create tasks and they are payed for by facebook; but we can test out if the feature works well by creating our own decentralized tasks). I was thinking this should be included for now to make it easy to test without affecting normal behavior, and we can remove optional decentralized eaas when we need to?